### ISSUE TITLE
Rule: 'region' - <section> element with non-empty 'aria-label' or 'aria-labelledby' or 'title' attributes is not being detected as a landmark

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#640]
```
<section aria-label="name1">
        <h1>Section-1: With aria-label</h1>
        <p>Content 1: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua</p>
    </section>

    <section aria-labelledby="name2">
        <h1 id="name2">Section-2: With aria-labelledby</h1>
        <p>Content 2: Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat</p>
    </section>

    <section title="name3">
        <h1>Section-3: With title</h1>
        <p>Content 3: Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur</p>
    </section>
```

**Comment by iamrafan** [comment]
thanks @WilcoFiers
